### DISCUSSION TITLE
Add multiple column sort to control panel

### DISCUSSION THREAD

**Discussion by matthewstick** [craftcms/cms#14697] [Ideas]
Hello,

Clients as for this a lot.  Ability to click on more than one column in the craft cms control panel to sort.  Also, ability to set more than one column for sorting in the configuration.

You can see how AGGRID does it here: https://ag-grid.com/example/  Just hold down shift when you click a column, and it will sort by a second column, third, etc.  You can click twice to change the direction. 

Thanks!

**Comment by brandonkelly** [comment]
Element indexes do have multi-column sorting; it’s just not revealed in the UI. If you sort by Column 1 DESC, then Column 2 ASC, any elements with identical values in Column 2 will be sorted by Column 1 DESC. (See #10203)

**Comment by matthewstick** [comment]
I'm sorting by column 1 (alphabetical titles), then by column 2 (a check box).  And column 1 titles are all scrambled.

Regardless - the request is to please reveal in the UI multi column sort (perhaps like how AG grid elegantly does it by clicking shift).  And also ability to set more than one column for sorting in the configuration.

A client has a view of 120,00 entries, and they want to sort by date and title and then a lightbox.  Thats just one example.

Thanks!
